### ISSUE TITLE
Broken after TV updates

### ISSUE THREAD

**Issue by alexpunct** [timelyart/Kairos#1]
**Describe the bug**
Kairos stopped working after the last tradingview updates, I think it can't find the elements to click one, here's what it's saying:

```2019-02-16 00:09:44.270 ERROR tv - create_alert: Message:
Traceback (most recent call last):
  File "C:\Sites\Kairos\tv\tv.py", line 760, in create_alert
    wait_and_click(alert_dialog, css_selectors['dlg_create_alert_ringtone'])
  File "C:\Sites\Kairos\tv\tv.py", line 244, in wait_and_click
    ec.element_to_be_clickable((By.CSS_SELECTOR, css_selector))).click()
  File "C:\Python37-32\lib\site-packages\selenium-3.141.0-py3.7.egg\selenium\webdriver\support\wait.py", line 80, in until
    raise TimeoutException(message, screen, stacktrace)
selenium.common.exceptions.TimeoutException: Message:

2019-02-16 00:09:44.280 INFO tv - retry: Trying again (1)

**Comment by timelyart** [comment]
The Alert Dialog changed in such a way that the ringtone checkbox is hidden from view.

**Comment by timelyart** [comment]
I've checked all the 'Alert Actions' and made sure they are all working again. 

Fixed in [v1.2.7](https://github.com/timelyart/Kairos/releases/tag/v1.2.7)
